### ISSUE TITLE
Cursor doesn't move when using spacebar for linux app

### ISSUE THREAD

**Issue by pask81** [Taha-Firoz/virtual.keyboard.dart#4]
Hello I am not sure if this is the same issue as https://github.com/Taha-Firoz/virtual.keyboard.dart/issues/2 anyway the problem I saw is that the cursor doesn't move when using a spacebar in a linux app, the space is actually added, but it is necessary to manually click on the text field to move the cursor. I use the same code for android/ios/web/linux app and this issue only happens for linux

**Comment by certosadipavia** [comment]
The problem appears for web as well.
When typing the characters: 
a`space`b`space`c

the TextField shows abc

and the string returned by the controller of the TextField is
abc`space` `space`

**Comment by Taha-Firoz** [comment]
Yup, it was a bug in how I was handling space bars, I've made a fix for it, I also merged a PR that might fix it. Sorry for the wait guys!

**Comment by athvaith8282** [comment]
Hey Is the spacebar bug is fixed ? , I have tried the package, Looks like the spacebar bug is still there !

**Comment by Taha-Firoz** [comment]
Can you try pulling the main branch and trying it from it instead of the pub.dev package?

I need to get around to publishing the fix for it.

**Comment by athvaith8282** [comment]
I tried from the main branch only, The spacebar didn't work.
